2-mandatory/4-people-i-know.js

```diff
@@ -377,65 +377,75 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
+let thirtyFiveOrOlder = friends.filter(friend=> friend.age >= 35).map(h=> h.name.first);
```

Comment:
You could be more descriptive inside your map function and give better names instead of `h` maybe you could use friend?

Is not a big deal but you know!!!1